### ISSUE TITLE
`failed to unpack truncated message`

### ISSUE THREAD

**Issue by aeijdenberg** [govau/sdget#2]
After around 8 medium size TXT records, I received this error:

```
Error doing DNS lookup:
error executing DNS query: dns: failed to unpack truncated message
```

When using `dig xxx TXT`, it showed:

```
;; Truncated, retrying in TCP mode.

...

;; MSG SIZE  rcvd: 517
```

and then seemed to work as expected.

**Comment by sarneaud** [comment]
`;; Truncated, retrying in TCP mode.`
Yep, sdget is currently using https://github.com/miekg/dns, which doesn't support TCP.

Options:
* Add TCP to the go library
* Use another library

**Comment by sarneaud** [comment]
Actually, it looks like it does support TCP after all, but doesn't do an automatic failover.

**Comment by sarneaud** [comment]
Hi, there's a fix for this in the new v0.4.0 release.
